tensorboard/data/server/run.rs

```diff
@@ -269,8 +310,12 @@ fn read_event(
         }
         Some(wt) => wt,
     };
-    if start_time.map(|start| wall_time < start).unwrap_or(true) {
-        *start_time = Some(wall_time);
+    if data
+        .start_time
+        .map(|start| wall_time < start)
+        .unwrap_or(true)
```

Comment:
Can be `.map_or(true, |start| wall_time < start)`? (Could have been
originally, but presumably I forgot; now, it makes the code short enough
to not have to spill onto four lines, so worth changing imho.)